config/deployments/production.toml

```diff
@@ -101,6 +101,7 @@ opennode.base_url = "https://api.opennode.com"
 paybox.base_url = "https://ppps.paybox.com/PPPS.php"
 paybox.secondary_base_url="https://tpeweb.paybox.com/"
 payeezy.base_url = "https://api.payeezy.com/"
+payload.base_url = "https://api.payload.com"
```

Comment:
production endpoint will be same. the environment will be detected by the api key (test environment have `test_` added to the key)